python/ray/tests/test_exit_observability.py

```diff
@@ -172,11 +174,10 @@ def verify_exit_code_0():
         type = worker["exit_type"]
         detail = worker["exit_detail"]
         assert type == "INTENDED_USER_EXIT" and "exit code 0" in detail
-        return verify_failed_task(
-            name="exit_with_exit_code",
-            error_type="ACTOR_DIED",
-            error_message=["INTENDED_USER_EXIT", "exit code 0"],
-        )
+        t = list_tasks(filters=[("name", "=", "exit_with_exit_code")])[0]
+        # exit_actor should be shown as non-task failure.
```

Comment:
this is not exit_actor? It sounds like exit(0)?

Reply:
yeah, this is exit(0) as well.